recipes/vir-simd/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class VirSIMDConan(ConanFile):
+    name = "vir-simd"
+    description = "This project aims to provide a simple fallback for users of std::experimental::simd (Parallelism TS 2)"
```

Comment:
How about "A fallback std::experimental::simd (Parallelism TS 2) implementation with additional features"

Reply:
Yes, much better. I just copied the first line from the repository.